### PR TITLE
Adding readme and license file, removing composer.lock

### PR DESCRIPTION
**Pull request by Kryniol**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,188 @@
...
+=================
+
+  * [Introduction](#introduction)
+  * [License](#license)
+  * [Getting started](#getting-started)
+  * [Usage](#usage)
+
+Introduction
+=================
+This Symfony3 bundle allows to validate whether an image (for instance uploaded by a user of your app) contains proper face.
```
Comment (blahy) [inline README.md]: contains proper face - this sounds weird to me, but im not a native so its difficult for me to propose something different.

contains image that shows person's face? Its not your face that is improper, but the image :)

File: README.md [hunk 2]
```diff
@@ -1,4 +1,188 @@
...
+}
+```
+
+In the configuration file you must provide your [MS Azure subscription key for Face API](https://azure.microsoft.com/en-us/try/cognitive-services/?api=face-api):
+
+```yml
+# app/config/config.yml
+
+xsolve_face_validator:
+    azure_subscription_key: your-subscription-key
```
Comment (blahy) [inline README.md]: if it is shown like that, wont it encourage people to put it into versioned config.yml?

File: README.md [hunk 3]
```diff
@@ -1,4 +1,188 @@
...
+// src/AppBundle/Entity/User.php
+
+use Symfony\Component\Validator\Constraints as Assert;
+use XSolve\FaceValidatorBundle\Validator\Constraints\Face;
+
+class User
+{
+    /**
+     * @var Symfony\Component\HttpFoundation\File\UploadedFile
+
```
Comment (blahy) [inline README.md]: no * in this line

File: README.md [hunk 4]
```diff
@@ -1,4 +1,188 @@
...
+```php
+// src/AppBundle/Entity/User.php
+
+use Symfony\Component\Validator\Constraints as Assert;
+use XSolve\FaceValidatorBundle\Validator\Constraints\Face;
+
+class User
+{
+    /**
+     * @var Symfony\Component\HttpFoundation\File\UploadedFile
```
Comment (blahy) [inline README.md]: you have some uses,but no uses for this for example. this is intended?

File: README.md [hunk 5]
```diff
@@ -1,4 +1,188 @@
...
+Assuming in your application you already have some model representing a user (for instance Doctrine entity) and it's
+used in a form to gather the data from users and perform validation, it already contains a profile picture:
+
+```php
+// src/AppBundle/Entity/User.php
+
+use Symfony\Component\Validator\Constraints as Assert;
+use XSolve\FaceValidatorBundle\Validator\Constraints\Face;
+
+class User
```
Comment (blahy) [inline README.md]: maybe example of valdiation if you only have the image (on disk for example) would be handy. I can see people employ this library for existing users

File: README.md (hunk 2 above)
Comment (Kryniol) [reply README.md]: the way people store their parameters is not the subject of this manual :)
also you shouldn't create your forms in controllers nor have your model fields exposed with visibility public but this way it's better readable (I took it from the Symfony docs)

File: README.md (hunk 5 above)
Comment (Kryniol) [reply README.md]: will add

File: README.md (hunk 4 above)
Comment (Kryniol) [reply README.md]: will add

File: README.md (hunk 1 above)
Comment (Kryniol) [reply README.md]: corrected

File: README.md (hunk 5 above)
Comment (Kryniol) [reply README.md]: added

File: README.md (hunk 3 above)
Comment (Kryniol) [reply README.md]: fixed

File: README.md (hunk 4 above)
Comment (Kryniol) [reply README.md]: added new example

**Pull request merged**